packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -6,6 +6,9 @@
       :cancel="closeQuotaModal"
       :spaces="selectedPersonalDrives"
       :max-quota="maxQuota"
+      :selected-resources="items"
```

Comment:
This can be removed, or am I missing something? 🤔

Reply:
Correct, dev leftovers